### PR TITLE
[react-loadable] Add server side rendering webpack plugin and getBundles helper function

### PR DESCRIPTION
**Pull request by iMobs**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/jamiebuilds/react-loadable#webpack-plugin>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-loadable/webpack.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  id: number;
+  name: string;
+  file: string;
+}
+
+export interface Manifest {
+  [moduleId: string]: Bundle[];
+}
+
+export function getBundles(manifest: Manifest, moduleIds: string[]): Bundle[];
```
Comment (Kovensky) [inline types/react-loadable/webpack.d.ts]: This file does commonjs exports; can't use ES module syntax here. https://github.com/jamiebuilds/react-loadable/blob/master/src/webpack.js#L62-L63

```ts
const exports: {
  getBundles: typeof getBundles;
  ReactLoadablePlugin: typeof ReactLoadablePlugin;
}
export = exports
```

File: types/react-loadable/webpack.d.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import * as webpack from 'webpack';
```
Comment (Kovensky) [inline types/react-loadable/webpack.d.ts]: webpack is a commonjs module; import it like a commonjs module.

`import webpack = require('webpack')`

**Review by Jessidhia — CHANGES_REQUESTED** [review]

File: types/react-loadable/test/webpack.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import * as webpack from 'webpack';
+import { ReactLoadablePlugin, getBundles } from 'react-loadable/webpack';
```
Comment (Kovensky) [inline types/react-loadable/test/webpack.ts]: These also should be commonjs imports

```ts
import webpack = require('webpack');
import { ReactLoadablePlugin, getBundles } = require('react-loadable/webpack');
```

**Review by Jessidhia — COMMENTED** [review]
I thought the new dtslint was going to catch these; or is it only if the tsconfig has esModuleInterop? 🤔

**Review by Jessidhia — APPROVED** [review]
:rocket:

File: types/react-loadable/test/webpack.ts (hunk 3 above)
Comment (iMobs) [reply types/react-loadable/test/webpack.ts]: Cool with the lack of destructuring? Doesn't seem possible.

**Review by Jessidhia — APPROVED** [review]

File: types/react-loadable/test/webpack.ts (hunk 3 above)
Comment (Kovensky) [reply types/react-loadable/test/webpack.ts]: Sure, probably better that way as well.

**Pull request merged**